ci/cpu/gcc12_release_cxx20.yml

```diff
@@ -4,6 +4,7 @@ include:
 cpu gcc12 cxx20 release deps:
   extends: .build_deps_common
   variables:
+    EXTRA_APTGET: "g++-12"
```

Comment:
I would add `gfortran-12` as well

Reply:
I'm going to give a check and I'll report here 😉